mabl snippets/isElementPresent.js

```diff
@@ -13,14 +12,18 @@ function mablJavaScriptStep(mablInputs, callback) {
     
     // conditional statement to assure that the element is present, as well as
     // assuring the display does not equal 'none'
-    if (element) {
-        const style = window.getComputedStyle(element);
-        if (style && style.display !== 'none' && style.visibility !== 'collapse' && style.visibility !== 'hidden') {
-             callback('Element Present');
+    if (!element) {
+        throw Error('Element Not Present');
+        
+     // If element is present, check display and visibility values to determine if element is present
+     } else {
```

Comment:
you don't need the else here. If there is no element, and we throw an error, it won't continue to the code below.

Reply:
Makes sense, removed the else statement